#### PLC connection stuff
ip: "192.168.1.10"
port: 4545

#### CTT Voltage Reference
Reference_Voltage: 20

#### Controller rate ####
freq: 20. # (Hz)

# k_rho, k_alpha, kv_alpha, ks, kv, sat_lat, sat_long,\
#                  waypoints, stop_limit, PSback_X, PSback_Y, PSback_yaw, length,\
#                  dock_X, dock_Y, dock_yaw

# Point Stabilization Control
k_rho: 7
k_alpha: 0.05
kv_alpha: 50
PSback_X: 685552.00531 # 685555.1928423041 #685556.648 
PSback_Y: 9204398.54478 # 9204399.226095065 #9204401.395
PSback_yaw: 0.773214 # 0.7988110232672496 #0.7993
length: 11.92 

# Docking control
dock_X: 685540.485170
dock_Y: 9204386.794055
dock_yaw: 0.773214 #0.7993

# Undockind control
undock_X: 685556.648 
undock_Y: 9204401.395
undock_yaw: 0.7943453892182503

# Stanley Control
ks: 5
kv: 1

# signal saturation value
sat_lat_max: 0.785 #rad
sat_lat_min: -0.785 #rad
sat_long_max: 1.5 #1.5 #m/s

# Stop_limit, distance to last waypoints
stop_limit: 3

# watpoints_path
waypoints_path: 'sigmoid220722.npy'

# Rosbag script bash
record_script: 'rosbag_record.sh'
record_folder: 'rosbag/'